templates/board.html

```diff
@@ -3,6 +3,8 @@
 
 <head>
   <meta charset="UTF-8">
+  <!-- refreshes every 6 min -->
+  <meta http-equiv="refresh" content="360" />
```

Comment:
Is this just to get updates when we push them? If so, can we make it longer like every 20min? Just so that it's less likely people walk up when videos are being refreshed.